pkg/destinations/bigquery/bigquery.go

```diff
@@ -0,0 +1,64 @@
+package bigquery
+
+import (
+	"context"
+
+	"cloud.google.com/go/bigquery"
+	"github.com/rs/zerolog/log"
+	"github.com/scratchdata/scratchdata/util"
+	"golang.org/x/oauth2/google"
+	"google.golang.org/api/option"
+)
+
+type BigQueryServer struct {
+	DatasetID             string `mapstructure:"bigquery_dataset_id"`
```

Comment:
Let's remove this and have the user specify a full table (`dataset_name.table_name`) when they call the POST endpoint. If the dataset doesn't exist, we should just create it for them.

Reply:
Not sure.

Streaming relies on it, other parts are coverable through sql, not sure how to make this query
 
![Screenshot from 2024-03-18 22-52-46](https://github.com/scratchdata/scratchdata/assets/47126953/2987799d-dc32-4205-9b60-b254191a6f16)

Reply:
How about parsing the `table` param? `strings.Split(table, ".")`

Reply:
Ignore, got it

Reply:
The last link works